### PR TITLE
Refs #19305 - fixing spacing issues on facts page

### PR DESCRIPTION
**Pull request by lizagilman**
image

### CODE REVIEW

File: app/helpers/fact_values_helper.rb [hunk 1]
```diff
@@ -8,19 +8,17 @@ def fact_from(record)
   def fact_name(value, parent)
     value_name = name = h(value.name)
     memo       = ''
-    name       = name.split(FactName::SEPARATOR).map do |current_name|
+    name.split(FactName::SEPARATOR).map do |current_name|
       memo = memo.empty? ? current_name : memo + FactName::SEPARATOR + current_name
       content_tag(:li) do
-        create_fact_name_link(parent, current_name, params[:host_id], value, value_name, memo)
+        if value.compose and current_name == name.split(FactName::SEPARATOR).last
```
Comment (timogoebel) [inline app/helpers/fact_values_helper.rb]: Please use `&&` instead of `and`.

**Review by timogoebel — CHANGES_REQUESTED** [review]

**Comment by timogoebel** [comment]
Looks good now:
image

**Comment by lizagilman** [comment]
@timogoebel updated

**Comment by lizagilman** [comment]
[test katello]

**Review by dLobatog — APPROVED** [review]
screenshot from 2017-08-07 11-03-13

screenshot from 2017-08-07 11-04-07


I think I'm going to merge @timogoebel , hopefully I'm not missing anything else. Thanks @lizagilman & @timogoebel

**Pull request merged**